### DISCUSSION TITLE
Hypergamy and Crossover Cells That Make Organism Crossover With Target

### DISCUSSION THREAD

**Discussion by theomgdev** [MaxRobinsonTheGreat/EvolutionSimulatorV2#13] [Ideas and Suggestions]
If an eye-like cell sees another organism with a longer-existing crossover cell(or just some kind of fitness criteria), it triggers movement towards it, and when it hits, a random exchange of parts occurs, covering half of the cells. An organism that has crossover once cannot crossover again.

**Comment by MaxRobinsonTheGreat** [comment]
This would allow for crossovers from wildly different species, and I would prefer a more selective breeding method.
